schemas/test/en/test_introduction.json

```diff
@@ -239,6 +252,430 @@
                                 }
                             ]
                         },
+                        {
+                            "type": "Question",
+                            "id": "report-radio",
+                            "question": {
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "Please provide figures for the period in which you were trading."
+                                        }
+                                    ]
+                                },
+                                "description": ["<p>Your return should relate to the calendar year 2021.</p>"],
```

Comment:
Testing on a local runner, the question says `Are you able to report for the calendar month 1 May 2016 to 31 May 2016?` is it meant to ?
Also when I press `Yes `I get directed to `Please provide optional start date`.